src/connector.ts

```diff
@@ -612,12 +613,14 @@ export async function connect(
         await vscode.workspace.fs.stat(portFile);
         const bytes = await vscode.workspace.fs.readFile(portFile);
         port = new TextDecoder('utf-8').decode(bytes);
+        outputWindow.appendLine(`; Read port file: ${portFile}`);
```

Comment:
To be consistent with the "; Using host:post ..." statement below, maybe this could be placed before the file is read and say "; Reading port file: ..."?

Reply:
I agree. Will fix!